csrankings-s.csv

```diff
@@ -1832,6 +1833,7 @@ Sui-Tung Mak,University of Southampton,https://www.ecs.soton.ac.uk/people/tsm3g1
 Suining He,University of Connecticut,http://hesuining.weebly.com,3OlOXn4AAAAJ
 Suiping Zhou,Middlesex University,http://www.cs.mdx.ac.uk/people/suiping-zhou,zhbtLUQAAAAJ
 Sujata Pal,IIT Ropar,http://www.iitrpr.ac.in/cse/sujata,9gcmeKwAAAAJ
+Sujay Deb,IIIT Delhi,https://www.iiitd.ac.in/sdeb,e3z7tT0AAAAJ

```

Comment:
ECE, not CS. Please explain how they meet the inclusion criteria.

Reply:
Dear Prof. Berger,

Dr. Sujay is with both ECE and CSE. While his profile details did mention that, our web admin had wrongly labeled him as just ECE faculty. We have now got his webpage corrected. 

Please let me know if you have any further concerns.    

-KR